### ISSUE TITLE
Some HTML character codes copied to clipboard

### ISSUE THREAD

**Issue by dmorgan-fa** [allmarkedup/lookbook#59]
Testing this against `v0.5.1`

Click `copy to clipboard` has some HTML characters when using an `erb` example:

Expected:
```
<%= render(CardComponent.new) do |c| %>
  <% c.header do %>
    <h2 class="fe-u-heading--4">Heading</h2>
  <% end %>
  <% c.row do %>
    This content is in a row.
  <% end %>
  <% c.footer(divider: divider) do %>
    Footer content goes here.
  <% end %>
<% end %>
```

Result:
```
&lt;%= render(CardComponent.new) do |c| %&gt;
  &lt;% c.header do %&gt;
    <h2 class="fe-u-heading--4">Heading</h2>
  &lt;% end %&gt;
  &lt;% c.row do %&gt;
    This content is in a row.
  &lt;% end %&gt;
  &lt;% c.footer(divider: divider) do %&gt;
    Footer content goes here.
  &lt;% end %&gt;
&lt;% end %&gt;
```

**Comment by allmarkedup** [comment]
Hey @dmorgan-fa, thanks for the bug report and sorry for the very slow reply! It's been a busy start to the year.

I've tested this and am seeing this behaviour too. It's a very odd one to be honest, I'm not quite sure where the character encoding is happening but I'll try to dig into it today and let you know when I've got a fix for it.

**Comment by allmarkedup** [comment]
@dmorgan-fa this should now be fixed in v0.5.2 but let me know if you still have any issues with it after updating. Thanks again for the bug report!

**Comment by dmorgan-fa** [comment]
Hey @allmarkedup, no worries, thanks for resolving this!
I've just upgraded, and it's looking good for us.
